### PR TITLE
Removed quotation marks from titles, corrected typo

### PR DESCRIPTION
**Pull request by rpetrusha**
# Removed quotation marks from titles, corrected typo

## Suggested Reviewers

@mairaw

### CODE REVIEW

File: docs/framework/migration-guide/runtime/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+--
```
Comment (mairaw) [inline docs/framework/migration-guide/runtime/index.md]: this change breaks the topic. you should add the 3rd dash back

**Review by mairaw — APPROVED** [review]
LGTM modulo the metadata block issue

**Comment by mairaw** [comment]
I'll merge this. BTW, I just noticed this was something I had addressed earlier here:
https://github.com/dotnet/docs/commit/97aa90b393871ccc2b6ad869a6528d140a94077f#diff-73405a6b7d1e83d9742692af873ae2da

and then it was introduced again in the following commit:
https://github.com/dotnet/docs/commit/73923bafaed2a64e66d2f663a2bb31a3869228bc#diff-73405a6b7d1e83d9742692af873ae2da

@conniey is it a tool that is adding the quotes?

/cc @rpetrusha

**Comment by conniey** [comment]
@mairaw @rpetrusha  Yes... it was the tool adding those in. I don't check all the regenerated breaking changes before submitting a commit. Can we compile a list of issues so I can look through them all and fixing the tool?

**Comment by rpetrusha** [comment]
@conniey, I'll review the PRs that Maira submitted to fix migrated compat topics and review some of the migrated topics and send you a list of issues next week.

**Pull request merged**